docs/develop/tokenomics/generation-and-decay.mdx

```diff
@@ -0,0 +1,46 @@
+---
+SPDX-License-Identifier: Apache-2.0
+sidebar_label: "Generation and Decay"
+---
+
+# DUST generation & decay
+
+This page defines the conceptual mechanics for how DUST supply is created and removed. Parameters are governed and may change over time. Tutorials reference this page to explain expected behaviors without assuming specific values.
+
+## Concepts
+
+- **Designation:** NIGHT holdings that are eligible to generate DUST (per protocol rules).
+- **Generation:** DUST accrues as a function of designated NIGHT and governed rates.
+- **Decay:** DUST balances **decrease continuously** by a governed decay function to prevent hoarding and stabilize fee markets.
+- **Consumption:** Executing transactions **spends** (burns) DUST.
+- **Caps / Safeguards:** A per-NIGHT cap may bound effective generation to reduce outlier effects.
+
+## High-level behavior
+
+- Let `G(t)` denote the **generation rate** at time `t`, governed.
+- Let `D(t)` denote **decay** over time `t`, governed (e.g., exponential or piecewise-linear).
+- **Effective DUST** at time `t2` from designated NIGHT amount `N` starting at `t1` is:
+
+```
+DUST(t2) = ∫[t1→t2] ( N × G(t) ) dt - ∫[t1→t2] ( Decay(DUST, t) ) dt - Consumption
```

Comment:
This is really overcomplicating it! See this document for details on the actual function: https://github.com/midnightntwrk/midnight-ledger/blob/main/spec/dust.md#design-overview